### ISSUE TITLE
AVT 1 - ComboBox has DAP violations 

### ISSUE THREAD

**Issue by icemanlily** [carbon-design-system/carbon#4415]
Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v9 - React
DAP Version 1.8.0.0 - IBM Accessibility WCAG 2.1 Sept. 2019 Ruleset

Description
Go to the React ComboBox component
Start DAP and the following violations are present (see screenshot):

![image](https://user-images.githubusercontent.com/6970471/67256121-2ac19400-f4b8-11e9-8e07-dd201f707339.png)

**Comment by dakahn** [comment]
Can you make the accessible name match the visible label?

**Comment by icemanlily** [comment]
@dakahn I removed aria-label="open menu" and the issue is missing. Could you help try it?

**Comment by dakahn** [comment]
Can the `aria-label` be made to match the visible label thus eliminating the error?

**Comment by icemanlily** [comment]
@dakahn I'm not sure what should  the aria-label be. But 'open menu' should be incorrct.

**Comment by icemanlily** [comment]
This issue should be fixed in Carbon V10. So close this issue.
